packages/eas-cli/src/submissions/archiveSource/ArchiveFileSource.ts

```diff
@@ -96,9 +101,9 @@ async function handleUrlSourceAsync(source: ArchiveFileUrlSource): Promise<Resol
 
 async function handleLatestSourceAsync(source: ArchiveFileLatestSource): Promise<ResolvedArchive> {
   try {
-    const artifactUrl = await getLatestBuildArtifactUrlAsync(source.platform, source.projectId);
+    const buildDetails = await getLatestBuildInfoAsync(source.platform, source.projectId);
```

Comment:
Please be consistent with naming. You're using multiple names for the same thing: build details, build info. Generally, I'd choose just `build`.

Reply:
Yes, you're right - I renamed everything to `build` except the summary field, which I called `formattedBuild`, because it's a `string`.